aws_analysis_tools/ec2_events/__init__.py

```diff
@@ -1 +0,0 @@
-from ec2_events import Application, main
```

Comment:
The tests pass(tm), and I tried each of the CLI commands/entry points, so I think it is safe to delete.